docs/tutorial/node/src/getting_started.md

```diff
@@ -0,0 +1,44 @@
+# Getting Started
+
+In this tutorial, we use JavaScript as the host programming language. We also support other programming languages like
+[Rust](https://slint-ui.com/docs/rust/slint/) or [C++](https://slint-ui.com/docs/cpp/).
+
+You will need a development environment with [node.js 16](https://nodejs.org/download/release/v16.19.1/) and [npm](https://www.npmjs.com/) installed. More recent
+versions of NodeJS are currently not supported, check [Issue #2220](https://github.com/slint-ui/slint/issues/2220).
+
+We're going to use `slint-ui` as `npm` dependency.
+
+In a new directory, we create a new `package.json` file.
+
+```json
+{{#include package.json}}
+```
+
+This should look familiar to people familiar with nodejs. We see that this package.json
```

Comment:
I'm not sure though - we use NodeJS ourselves, here it's nodejs and on the website they write Node.js. Either way I suppose we should pick one and use it consistently :)

```suggestion
This should look familiar to people familiar with NodeJS. We see that this package.json
```